t/50chopblanks.t

```diff
@@ -26,42 +26,59 @@ ok $dbh->do("DROP TABLE IF EXISTS dbd_mysql_t50chopblanks"), "drop table if exis
 my $create= <<EOT;
 CREATE TABLE dbd_mysql_t50chopblanks (
   id INT(4),
-  name TEXT
+  c_varchar VARCHAR(10),
```

Comment:
VARCHAR is problematic. MySQL server can configured to automatically chop blanks. We could expects that this test start randomly on cpan failing. See commit: ce14f6c989eb3e626586a24aed16ee6ebcf18a82